packages/components/src/FormField/FormField.css

```diff
@@ -66,6 +66,13 @@
   box-shadow: var(--shadow-focus);
 }
 
+.horizontalWrapper {
```

Comment:
I don't love this name. it's not very descriptive

container, wrapper, inputWrapper, childrenWrapper and a couple other names were already taken was the problem but there's gotta be something better I can come up with